src/plugins/utils/LambdaEventUtils.ts

```diff
@@ -360,6 +363,32 @@ class LambdaEventUtils {
         return className;
     }
 
+    static injectTriggerTagsForNetlify(span: ThundraSpan, pluginContext: any, originalContext: any): string {
+        const className = ClassNames.NETLIFY;
+        const domainName = DomainNames.API;
+        let operationName = 'netlify-site';
+
+        const siteName = process.env[NetlifyConstants.NETLIFY_SITE_NAME];
```

Comment:
It is better to get env var over `Utils.getEnvVar` instead of dealing with `process.env` directly